\doxysection{Ramsey\textbackslash{}Uuid\textbackslash{}Exception\textbackslash{}Random\+Source\+Exception Class Reference}
\hypertarget{class_ramsey_1_1_uuid_1_1_exception_1_1_random_source_exception}{}\label{class_ramsey_1_1_uuid_1_1_exception_1_1_random_source_exception}\index{Ramsey\textbackslash{}Uuid\textbackslash{}Exception\textbackslash{}RandomSourceException@{Ramsey\textbackslash{}Uuid\textbackslash{}Exception\textbackslash{}RandomSourceException}}
Inheritance diagram for Ramsey\textbackslash{}Uuid\textbackslash{}Exception\textbackslash{}Random\+Source\+Exception\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=2.828283cm]{class_ramsey_1_1_uuid_1_1_exception_1_1_random_source_exception}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
Thrown to indicate that the source of random data encountered an error

This exception is used mostly to indicate that random\+\_\+bytes() or random\+\_\+int() threw an exception. However, it may be used for other sources of random data. 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
vendor/ramsey/uuid/src/\+Exception/Random\+Source\+Exception.\+php\end{DoxyCompactItemize}
